src/base/types/Isozyme.jl

```diff
@@ -0,0 +1,17 @@
+"""
+    mutable struct Isozyme
+
+Struct containing isozyme information. Here, `stoichiometry` is a 
+dictionary of gene ids to their stoichiometry in the isozyme complex,
+and `kcats` is a tuple of the forward and reverse kcats of the isozyme.
```

Comment:
documentation: if `kcats` is an acronym it would be great to have it expanded at least once

Reply:
added the word turnover to the description